### PR TITLE
Widget fullscreen

### PR DESCRIPTION
**Pull request by cshuenss**
Make widgets fullscreenable.

- [x] Make modal go to top of screen when opened
- [x] Make graph scale with modal size
- [x] Extend to other widgets?

### CODE REVIEW

**Comment by cshuenss** [comment]
resolves #180

**Comment by George-Ogden** [comment]
I'll have a look at this

**Comment by George-Ogden** [comment]
@cshuenss I now get both charts to render fullscreen. There's an issue with the map but when I figure out what that is, we might be able to extend it.

**Comment by George-Ogden** [comment]
I think this would be very useful to have on the map/table

**Comment by cshuenss** [comment]
Thanks for the help. Will work on map/table fullscreening in the evening

**Comment by George-Ogden** [comment]
This looks good - can you update please and I'll approve

**Comment by cshuenss** [comment]
done @George-Ogden

File: devfiles/src/ui/OptionsComp/PanelOptionsComp.tsx [hunk 1]
```diff
@@ -22,17 +25,39 @@ function PanelOptionsComp(params: { pageManager: PageManager }) {
...
         <div key={uuidv4()}>
             <button
                 className='delete-btn'
                 onClick={() => {
-                    document.getElementById('delete_panel_modal').style.display = 'block';
-                    document.getElementById('delete_modal_name').innerText = params.pageManager
-                        .getSelectedPanel()
-                        .getName();
-                    document.getElementById('delete_modal_btn').onclick = () => {
+                    console.log('asf');
```
Comment (George-Ogden) [inline devfiles/src/ui/OptionsComp/PanelOptionsComp.tsx:33]: Is this meant to be here?

**Review by George-Ogden — APPROVED** [review]
Is there an easy way to increase the number of rows in the table on fullscreen?

**Review by George-Ogden — CHANGES_REQUESTED** [review]
There's the console.log issue. Is there a way to make the table display more rows in fullscreen?

**Comment by cshuenss** [comment]
I did think about increasing table row count, but each row can be a variable height

The default of 10 works for widgets, could just make it 15 or something if fullscreen

**Comment by George-Ogden** [comment]
> I did think about increasing table row count, but each row can be a variable height
> 
> The default of 10 works for widgets, could just make it 15 or something if fullscreen

I think that would be good. I also think 8 (or even 5) in a panel might be better so it looks less squashed.

**Comment by George-Ogden** [comment]
Don't worry about the rows - I'll do it in #255

**Review by George-Ogden — APPROVED** [review]

**Comment by cshuenss** [comment]
@George-Ogden just about to push the rows commit

**Comment by George-Ogden** [comment]
> @George-Ogden just about to push the rows commit

That looks really good - I'm happy to keep it

**Comment by cshuenss** [comment]
I'm busy until ~2pm so if there's any things you want to tweak and other things depend on it, it might be best to just make those changes while I'm not around

**Pull request merged**